ledger/ledger-api-common/src/test/suite/scala/com/digitalasset/platform/server/api/services/grpc/GrpcHealthServiceSpec.scala

```diff
@@ -3,22 +3,20 @@
 
 package com.digitalasset.platform.server.api.services.grpc
 
+import com.digitalasset.grpc.GrpcException
 import com.digitalasset.grpc.adapter.server.rs.MockServerCallStreamObserver
+import com.digitalasset.ledger.api.health._
 import com.digitalasset.ledger.api.testing.utils.AkkaBeforeAndAfterAll
+import com.digitalasset.platform.server.api.services.grpc.GrpcHealthService._
 import com.digitalasset.platform.server.api.services.grpc.GrpcHealthServiceSpec._
 import io.grpc.health.v1.health.{HealthCheckRequest, HealthCheckResponse}
 import org.scalatest.concurrent.Eventually
 import org.scalatest.time.{Second, Span}
 import org.scalatest.{Matchers, WordSpec}
 
+import scala.concurrent.duration.DurationInt
 import scala.concurrent.{Await, ExecutionContext}
 
-object GrpcHealthServiceSpec {
-  private val request = HealthCheckRequest()
-
-  private val servingResponse = HealthCheckResponse(HealthCheckResponse.ServingStatus.SERVING)
-}
-
 final class GrpcHealthServiceSpec
     extends WordSpec
```

Comment:
You can probably make an `AsyncWordSpec` out of this since most of the test have to deal with `Future`s. This would allow you to drop a log of `Await.result`.

Reply:
And you could also drop the `executionContext`, which would already be in scope for the test.

Reply:
This fell off my to-do list. Cheers for the reminder!